### PR TITLE
api: metric: lxd_cpu_seconds_total: fix help text, floats

### PR DESCRIPTION
**Pull request by exsver**
Pull request for the issue: https://github.com/lxc/lxd/issues/10820

### CODE REVIEW

**Review by tomponline — APPROVED** [review]

**Comment by tomponline** [comment]
jenkins: test this please

File: lxd/instance/drivers/driver_lxc.go [hunk 1]
```diff
@@ -6919,8 +6919,8 @@ func (d *lxc) Metrics() (*metrics.MetricSet, error) {
 		for cpu, stats := range usage {
 			cpuID := strconv.Itoa(int(cpu))
 
-			out.AddSamples(metrics.CPUSecondsTotal, metrics.Sample{Value: float64(stats.System / 1000000000), Labels: map[string]string{"mode": "system", "cpu": cpuID}})
-			out.AddSamples(metrics.CPUSecondsTotal, metrics.Sample{Value: float64(stats.User / 1000000000), Labels: map[string]string{"mode": "user", "cpu": cpuID}})
+			out.AddSamples(metrics.CPUSecondsTotal, metrics.Sample{Value: float64(stats.System) / float64(1000000000), Labels: map[string]string{"mode": "system", "cpu": cpuID}})
+			out.AddSamples(metrics.CPUSecondsTotal, metrics.Sample{Value: float64(stats.User) / float64(1000000000), Labels: map[string]string{"mode": "user", "cpu": cpuID}})
```
Comment (simondeziel) [inline lxd/instance/drivers/driver_lxc.go:6923]: Small nit: would you mind dropping the `float64()` around the constant?

Comment (exsver) [reply lxd/instance/drivers/driver_lxc.go]: Done

**Review by tomponline — APPROVED** [review]

**Comment by tomponline** [comment]
jenkins: test this please

**Review by simondeziel — APPROVED** [review]

**Comment by stgraber** [comment]
jenkins: test this please

**Pull request merged**